.editorconfig

```diff
@@ -13,6 +13,6 @@ trim_trailing_whitespace = false
 
 [*.java]
 # IDE specific config should not be included, but this workaround is required
-# to allow correct indentation of continued lines, otherwise an ident of two is
+# to allow correct indentation of continued lines, otherwise an indent of two is
```

Comment:
Could you submit this fix to the template too please.

Reply:
ok sure. I will